docusaurus/docs/learn/quickstarts/zac/index.md

```diff
@@ -86,6 +86,10 @@ you can perform the following steps.
    LISTEN 0      511                *:1408             *:*    users:(("node",pid=26013,fd=18))
    ```
 
+## Kubernetes
```

Comment:
Let's place this after the docker stuff at the end, not in the middle of it. I feel like in order of approachability running it bare as a dev -> using docker -> using compose -> using kubernetes makes sense